.eslintrc.js

```diff
@@ -18,7 +18,6 @@ module.exports = {
     'import/prefer-default-export': 'off',
     'simple-import-sort/imports': 'warn',
     quotes: [2, 'single', { avoidEscape: true }],
-    'prettier/prettier': ['error', { singleQuote: true }],
```

Comment:
We don't want to remove the rule entirely, just the options. Without options specified, Prettier run via ESLint will use the options configured in `.prettierrc.json`.

```
prettier/prettier': 'error',
```